lib/Elastica/Param.php

```diff
@@ -138,9 +138,12 @@ public function setParams(array $params)
     public function addParam($key, $value)
     {
         if ($key != null) {
-            if (!isset($this->_params[$key])) {
-                $this->_params[$key] = array();
-            }
+            /**
```

Comment:
Please remove this note. Lets keep the logic the same in this PR, so I suggest to uncomment the below again (especially as the tests fail).